core/services/ocr2/delegate.go

```diff
@@ -343,23 +347,95 @@ func (d Delegate) ServicesForSpec(jobSpec job.Job) ([]job.ServiceCtx, error) {
 			runResults,
 			d.pipelineRunner,
 			make(chan struct{}),
-			lggr)
+			lggr,
+		)
 
 		// NOTE: we return from here with the services because the OCR2VRF oracles are defined
 		// and exported from the ocr2vrf library. It takes care of running the DKG and OCR2VRF
 		// oracles under the hood together.
 		oracleCtx := job.NewServiceAdapter(oracles)
 		return []job.ServiceCtx{runResultSaver, vrfProvider, oracleCtx}, nil
+	case job.OCR2Keeper:
+		chainIDInterface, ok := jobSpec.OCR2OracleSpec.RelayConfig["chainID"]
+		if !ok {
+			return nil, errors.New("chainID must be provided in relay config")
+		}
+		chainID := int64(chainIDInterface.(float64))
+		chain, err2 := d.chainSet.Get(big.NewInt(chainID))
+		if err2 != nil {
+			return nil, errors.Wrap(err2, "failed to get chainset")
+		}
+
+		var cfg ocr2keeperconfig.PluginConfig
```

Comment:
Just a thought: I'm not sure what all is included in the plugin config, but we may want to have some hardcoded part of the config which is fixed in all keepers. (e.g. what we did with observation source in previous spec)